src/qibolab/__init__.py

```diff
@@ -54,3 +55,27 @@ def create_platform(name, runcard=None):
     if runcard is None:
         return module.create()
     return module.create(runcard)
+
+
+def execute_qasm_circuit(
```

Comment:
```suggestion
def execute_qasm(
```

If you wish, QASM is enough :)